rustls-libssl/src/lib.rs

```diff
@@ -340,14 +343,22 @@ struct Ssl {
     alpn: Vec<Vec<u8>>,
     sni_server_name: Option<ServerName<'static>>,
     bio: Option<bio::Bio>,
-    conn: Option<Connection>,
-    verifier: Option<Arc<verifier::ServerVerifier>>,
+    conn: ConnState,
     peer_cert: Option<x509::OwnedX509>,
     peer_cert_chain: Option<x509::OwnedX509Stack>,
     shutdown_flags: ShutdownFlags,
     auth_keys: sign::CertifiedKeySet,
 }
 
+#[allow(clippy::large_enum_variant)]
+enum ConnState {
```

Comment:
Derive debug?

Reply:
Unfortunately `Acceptor` doesn't impl Debug, which makes Derive not possible. Going to leave this -- we could `impl fmt::Debug` manually later if that becomes useful in logging or something.